services/infra-cdk/lib/stacks/cognito.ts

```diff
@@ -243,40 +243,89 @@ export class CognitoStack extends BaseStack {
     }
 
     private createOutputs(): void {
+        // TEMPORARY: For dev/val/prod, export Serverless Cognito IDs from SSM
+        // For review environments, use the new CDK Cognito resources
+        //
+        // This allows:
+        // - dev/val/prod: Continue using existing Serverless Cognito (with Okta configured)
+        // - review envs: Use new CDK Cognito with test users (isolated per PR)
+        //
+        // Migration plan:
+        // 1. Now: Dev/val/prod envs use Serverless Cognito, review envs use CDK Cognito
+        // 2. Later: Update Okta to point to new CDK Cognito endpoints
+        // 3. Later: Change all envs to use CDK Cognito (remove conditional logic)
+        // 4. Later: Delete Serverless Cognito resources
+
+        const isDevValProd = ['dev', 'val', 'prod'].includes(this.stage)
```

Comment:
Are we confident these are the environment names (nothing capitalized?), just wondering this should be more flexible?